packages/fast-css-editor-react/app/app.tsx

```diff
@@ -1,212 +1,136 @@
-                        name={"CSS Position"}
-                        schema={CSSPositionSchema}
-                        component={CSSPosition}
+            <DesignSystemProvider designSystem={designSystemDefaults}>
+                <div
+                    style={{
+                        fontFamily:
+                            "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif",
+                    }}
+                >
+                    <div
+                        style={{
+                            minWidth: "250px",
+                            minHeight: "100vh",
+                            padding: "0 0 0 10px",
+                            boxSizing: "border-box",
+                            color: designSystemDefaults.foregroundColor,
+                            background: designSystemDefaults.backgroundColor,
+                            float: "left",
+                        }}
                     >
-                        <SiteCategoryDocumentation
-                            slot={"canvas-detail-view-documentation"}
-                        >
-                            <div>CSS Position</div>
-                        </SiteCategoryDocumentation>
-                        <SiteCategoryItem
-                            slot={"canvas-example-view"}
-                            data={Object.assign({}, this.state.positionExample, {
-                                onPositionUpdate: this.handlePositionExampleUpdate,
-                            })}
-                        />
-                        <SiteCategoryItem
-                            slot={"canvas-detail-view-example"}
-                            data={Object.assign({}, this.state.positionDetail, {
-                                onPositionUpdate: this.handlePositionDetailUpdate,
-                            })}
-                        />
-                    </SiteCategory>
-                    <SiteCategory
-                        slot={"category"}
-                        name={"CSS Spacing"}
-                        schema={CSSSpacingSchema}
-                        component={CSSSpacing}
+                        {this.renderCSSEditorComponent()}
+                    </div>
+                    <div
+                        style={{
+                            float: "left",
+                            marginLeft: "8px",
+                        }}
                     >
-                        <SiteCategoryDocumentation
-                            slot={"canvas-detail-view-documentation"}
+                        <div>
+                            <select onChange={this.handleComponentUpdate}>
+                                {this.renderOptions()}
+                            </select>
+                        </div>
+                        <pre
+                            style={{
+                                padding: "12px",
+                                background: "rgb(244, 245, 246)",
```

Comment:
Why are some hex and some rgb?

Reply:
No real reason, this is for testing so I just made it match some of our other values.